test/framework/toolchain.py

```diff
@@ -611,6 +611,7 @@ def setup_sandbox_for_intel_fftw(self, moddir, imklver='10.3.12.361'):
 
         fftw_libs = ['fftw3xc_intel', 'fftw3xc_pgi', 'mkl_cdft_core', 'mkl_blacs_intelmpi_lp64']
         fftw_libs += ['mkl_intel_lp64', 'mkl_sequential', 'mkl_core', 'mkl_intel_ilp64']
+        fftw_libs += ['mkl_intel_thread', 'iomp5', 'pthread']
```

Comment:
If compiler is GCC one should use
-lmkl_gf_lp64 -lmkl_gnu_thread -lmkl_core -lgomp -lpthread -lm -ldl
or
-lmkl_gf_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm -ldl
if using the intel openmp lib (not likely in the EB builds we have)

Is this taken care of somewhere else?

Reply:
Bah.... missed the"test/" part so can probably ignore...